### PR TITLE
Updated animation for when box is placed on column

### PR DESCRIPTION
**Pull request by kiranaatkar**

### CODE REVIEW

File: vr-wordle/src/Components/ColumnContainer.js [hunk 1]
```diff
@@ -1,30 +1,26 @@
...
 
-export default function ColumnContainer({
-  columnGeometry,
-  setGuess,
-  guessIndex,
-  material,
-}) {
+export default function ColumnContainer(props) {
+  const { columnGeometry, setGuess, guessIndex } = props;
   let [x, y, z] = columnGeometry;
```
Comment (tomw13) [inline vr-wordle/src/Components/ColumnContainer.js:6]: Like how you condensed this

File: vr-wordle/src/Components/PressurePlate.js [hunk 2]
```diff
@@ -1,24 +1,45 @@
...
-      e.body.name !== char && props.setGuess(e.body.name, props.guessIndex);
+    onCollide: (e) => {
+      props.setGuess(e.body.name, props.guessIndex);
+    },
+    onCollideBegin: () => {
+      setPress(true);
+    },
+    onCollideEnd: () => {
+      setPress(false);
     },
```
Comment (tomw13) [inline vr-wordle/src/Components/PressurePlate.js:22]: Really great work here

**Review by tomw13 — APPROVED** [review]
Looking good

File: vr-wordle/src/Components/PressurePlate.js [hunk 3]
```diff
@@ -1,24 +1,45 @@
...
+      <meshStandardMaterial color="black" />
+    );
+  }
+
   return (
-    <animated.mesh ref={ref} position={props.position}>
+    <animated.mesh ref={ref} scale={scale}>
       <cylinderBufferGeometry attach="geometry" args={props.args} />
-      <meshNormalMaterial color="hotpink" wireframe opacity={1} transparent />
+      {getPlateColor()}
```
Comment (TamoorW) [inline vr-wordle/src/Components/PressurePlate.js:42]: I like the use of functions here

**Review by TamoorW — APPROVED** [review]

**Pull request merged**